### PR TITLE
[CIVIS-9315] Update datascience-python to v8.0.0 to fix apt-get

### PR DESCRIPTION
**Pull request by mheilman**
This updates datascience-python to [v8.0.0](https://github.com/civisanalytics/datascience-python/releases/tag/v8.0.0) to fix `apt-get`.

[Here's](https://platform.civisanalytics.com/spa/#/notebooks/36253?expanded=true) an internal test notebook in Platform.

---

- [x] (For Civis employees only) Reference to a relevant ticket in the pull request title
- [x] Changelog entry added to `CHANGELOG.md` at the repo's root level
- [x] Description of change in the pull request description
- [x] If applicable, unit tests have been added and/or updated
- [x] The CircleCI builds have all passed

### CODE REVIEW

**Review by jacksonlee-civis — DISMISSED** [review]
LGTM

**Comment by mheilman** [comment]
Ope, I just realized this image has a separate requirements file. I'll update that.

File: Dockerfile [hunk 1]
```diff
@@ -2,9 +2,11 @@
 # Unfortunately Dockerhub isn't as flexible as CircleCi or AWS Codebuild, so we have to hardcode this value here.
 # So if you in the future need to update this value, make sure you also edit the value in .ds_python_version.
 # These values should be kept in sync.
-ARG DS_PYTHON_IMG_VERSION=7.3.0
+ARG DS_PYTHON_IMG_VERSION=8.0.0
 
-FROM civisanalytics/datascience-python:${DS_PYTHON_IMG_VERSION}
+ARG PLATFORM=linux/x86_64
```
Comment (mheilman) [inline Dockerfile:7]: I added this for consistency with datascience-python and civis-jupyter-r, to make it easier to test things locally on arm64 laptops

**Review by jacksonlee-civis — APPROVED** [review]
LGTM

**Pull request merged**